images/client/Containerfile

```diff
@@ -4,4 +4,4 @@ FROM registry.fedoraproject.org/fedora:36
 
 MAINTAINER Michael Adam <obnox@samba.org>
 
-RUN dnf -y install samba-client
+RUN dnf -y install samba-client systemd
```

Comment:
I would like to see a comment explaining the rationale for adding systemd. Otherwise, someone is likely to come along in the future and this it's strange/wrong and possibly spend time trying to understand it. :-)

Secondly, I probably missed this before, and thus I won't require it for an approval, but should we tack on a 'yum clean all' at the end to make sure dnf/rpm download artifacts are not left in the container image?

Reply:
I have added a comment and `dnf clean all` in the updated version.